hummingbot/client/command/rate_command.py

```diff
@@ -39,7 +38,7 @@ async def show_rate(self,  # type: HummingbotApplication
     @staticmethod
     async def oracle_rate_msg(pair: str,
                               ):
-        pair = pair.upper()
+        pair = pair.upper().strip('\"').strip("'")
```

Comment:
While we're at improving the preprocessing, it'd be good to have also strip() at the end to remove whitespaces.

-- Review submitted in representation of the Hummingbot Foundation Technical DAO

Reply:
Hi @MementoRC have you made this change yet?